__tests__/app.test.js

```diff
@@ -194,3 +195,24 @@ describe("4. 06-PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("5. 07-GET /api/articles/:article_id", () => {
+  test("status:200, responds with an article object by it's id with comment_count colum", () => {
+    const ARTICLE_ID = 1;
+    return request(app)
+      .get(`/api/articles/${ARTICLE_ID}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({

```

Comment:
toEqual objectContaining is more flexible for these situations, so you can add extra features later with their own tests, and still pass the original ones ... just something to look into for a refactor